setup.py

```diff
@@ -230,7 +230,7 @@ def is_macosx_sdk_path(path):
     Returns True if 'path' can be located in an OSX SDK
     """
     return ( (path.startswith('/usr/') and not path.startswith('/usr/local'))
-                or path.startswith('/System/')
+                or (path.startswith('/System/') and not path.startswith('/System/Volumes/Data'))
```

Comment:
As I suggested in an earlier comment on the bug tracker for this, alll we are interested in here are files in the SDK. For macOS 10.15 and 11 under /System, there are /System/Library and, for completeness, /System/iOSSupport. The test should be explicitly for those.

Reply:
Updated.